### ISSUE TITLE
Make some colours darker when using dark mode

### ISSUE THREAD

**Issue by AaronTechnic** [ubuntu/yaru#3660]
Hello Yaru devs!

I have been using Ubuntu 22.04 and the new accent colours are really nice. But I think you should make some of the colours, like blue and red, a bit darker, like Adwaita Dark. When using the blue accent scheme with a dark theme, the text highlighted with the blue accent colour looks a bit unreadable and stands out a lot. Compared to adwaita dark the blue colour is darker so it looks nice and doesn't stand out like that.

For colours like bark and sage, it's all fine, although I think changing all of the colours to make it darker in dark mode will make it look better.

What do you think?

**Comment by MilutinMujovic** [comment]
Yes, I agree with you. It is clear they focused on making them look good on light mode.

I would also suggest adding a yellow accent color since that's the only one from ROYGBIV that we're missing.

**Comment by AaronTechnic** [comment]
No update yet, but I hope this happens.
